app.js

```diff
@@ -31,6 +33,7 @@ app.post('/', function(request, response) {
     }
 
     // Assign issue to the next user in the queue and remove user from queue
+    const issueID = request.body.data.issue.id;
```

Comment:
Moved this down so that the server doesn't error early if there isn't a `data` or `issue` request body parameter.